src/server/qgsserverprojectutils.cpp

```diff
@@ -333,6 +333,55 @@ QStringList QgsServerProjectUtils::wmsOutputCrsList( const QgsProject &project )
   return crsList;
 }
 
+QStringList QgsServerProjectUtils::wmsOutputCrsListAsOgcUrn( const QgsProject &project )
+{
+  QStringList crsList;
+  const QStringList wmsCrsList = project.readListEntry( QStringLiteral( "WMSCrsList" ), QStringLiteral( "/" ), QStringList() );
+  if ( !wmsCrsList.isEmpty() )
+  {
+    for ( int i = 0; i < wmsCrsList.size(); ++i )
```

Comment:
a range-based for loop wold probably be more readable here
```suggestion
    for ( const auto &crs : std::as_const( wmsCrsList )
```

Reply:
This code does not exist anymore because this function now calls `wmsOutputCrsList` to avoid code duplication as suggested by @troopa81. However, I have added a commit to apply your suggestions to the existing  `wmsOutputCrsList` .